src/server.py

```diff
@@ -0,0 +1,45 @@
+"""functions to build the server side of echo."""
+
+
+from __future__ import unicode_literals
+import socket
+
+
+def server():
+    """Set up the server side to listen."""
+    server = socket.socket(
+        socket.AF_INET,
+        socket.SOCK_STREAM,
+        socket.IPPROTO_TCP
+    )
+    address = ('127.0.0.1', 5016)
+    server.bind(address)
+
+    server.listen(1)
+    print("Listening...")
+
+    conn, addr = server.accept()
+
+    while True:
+        try:
+            buffer_length = 8
```

Comment:
This is a great example of a value that can be made into a global constant, since the program does not change it.